apps/console/src/features/organizations/hooks/use-organization-switch.ts

```diff
@@ -79,9 +89,36 @@ const useOrganizationSwitch = (): UseOrganizationSwitchInterface => {
         return response;
     };
 
+    /**
+     * Handles the organization switch in the legacy mode.
+     * @param breadcrumbList - Breadcrumb list
+     * @param org - Organization to switch to.
+     */
+    const switchOrganizationInLegacyMode = (breadcrumbList: BreadcrumbList, org: GenericOrganization): void => {
+
+        let newOrgPath: string = "";
+
+        if (breadcrumbList && breadcrumbList.length > 0 && OrganizationUtils.isSuperOrganization(breadcrumbList[0])
+            && breadcrumbList[ 1 ]?.id === org.id && organizationConfigs.showSwitcherInTenants) {
+            newOrgPath = "/t/" + org.name + "/" + window[ "AppUtils" ].getConfig().appBase;
```

Comment:
Let's format this long conditional statement to improve readability.

```suggestion
        if (
            breadcrumbList?.length > 0 
            && OrganizationUtils.isSuperOrganization(breadcrumbList[0])
            && breadcrumbList[ 1 ]?.id === org.id &
            & organizationConfigs.showSwitcherInTenants
        ) {
            newOrgPath = "/t/" + org.name + "/" + window[ "AppUtils" ].getConfig().appBase;
```

Reply:
Resolved with 773dd41164aa3f63112cf9182d16c6d1e7908f37